- Install benchmark script with a less generic name
- Drop stringent setuptools version requirement
- Use of py-test-cov seems broken, so don't try.

Index: pyproject.toml
--- pyproject.toml.orig
+++ pyproject.toml
@@ -29,10 +29,10 @@ docs = "https://fastecdsa.readthedocs.io"
 pypi = "https://pypi.org/project/fastecdsa/"
 
 [project.scripts]
-benchmark = "fastecdsa.benchmark:run"
+fastecdsa-benchmark = "fastecdsa.benchmark:run"
 
 [build-system]
-requires = ["setuptools>=75.6.0"]
+requires = ["setuptools"]
 build-backend = "setuptools.build_meta"
 
 [tool.coverage.run]
@@ -43,7 +43,7 @@ omit = [
 
 [tool.pytest.ini_options]
 pythonpath = ["."]
-addopts = "--cov=fastecdsa --cov-report=term-missing --cov-fail-under=95"
+#addopts = "--cov=fastecdsa --cov-report=term-missing --cov-fail-under=95"
 
 [tool.setuptools]
 include-package-data = true
